wordpress-mobile/WordPress-iOS
# Navigation arrows for notifications detail

Introduces buttons to navigate to next/previous note from the notifications detail.

![simulator screen shot 6 feb 2017 12 08 04](https://cloud.githubusercontent.com/assets/8739/22644948/ef14e3a2-ec64-11e6-90ef-f7cec895127c.png)


Fixes #6140 

To test:

Go to a notification and navigate around. Some things to test:

- Buttons should be disabled when there are no next/previous notifications
- Notifications should be marked as read when you see them
- Getting a new notification while viewing the previously newest notification should update the navigation

@jleandroperez I'd love your opinion on the latest changes, but since you wrote most of this I'm asking @frosty to review it
cc @kwonye 
Needs review: @frosty
